spec/namespaces/indices.yaml

```diff
@@ -3477,9 +3477,10 @@ components:
     indices.forcemerge::query.max_num_segments:
       in: query
       name: max_num_segments
-      description: 'The number of segments the index should be merged into (default: dynamic)'
+      description: The number of segments the index should be merged into.
       schema:
         type: number
+        default: dynamic
```

Comment:
This one should just be reworded rather than specifying `default:` as it's not that the value of the field will be literally the string `dynamic` (as the schema type is number), it's just that the cluster will dynamically pick the number of segments it determines to be required/ideal. The docs https://opensearch.org/docs/latest/api-reference/index-apis/force-merge/#query-parameters describe the parameter as:
> The number of larger segments into which smaller segments are merged. Set this parameter to 1 to merge all segments into one segment. The default behavior is to perform the merge as necessary.